PNlib/Functions/OddsAndEnds/prioCheck.mo

```diff
@@ -0,0 +1,18 @@
+within PNlib.Functions.OddsAndEnds;
+function prioCheck "Determine the priotities are correct"
+  extends Modelica.Icons.Function;
+  input Integer enablingPrio[:] "enabling priorities";
+  input Integer N "number of transitions";
+  output Boolean result "= true, if priorities are Correctly";
+protected
+  Integer Index;  
+algorithm
+  Index:=1;
+  result:=true;
+  while result==true and Index<=N loop
+    if Modelica.Math.Vectors.find(Index,enablingPrio)==0 then
+        result:=false;
```

Comment:
I think it would be a bit nicer to use a return statement instead of the `result==true` condition for the while-loop.